### ISSUE TITLE
Removed files don't disappear from packs

### ISSUE THREAD

**Issue by ttencate** [eowise/gradle-packer-plugin#11]
If I remove an input file, but its output PNG still exists in `out/resources`, it still ends up in the final pack.

It would be great if this plugin could check if any PNGs in the intermediate directory correspond to an input file, and if not, remove them.

**Comment by acatinon** [comment]
I agree, it is a bit anoying. But it is not easy to know what output file is linked to a given deleted input file in all cases.

I'm still investigating...
